src/vmm/src/rpc_interface.rs

```diff
@@ -1065,6 +1067,7 @@ mod tests {
             path_on_host: String::new(),
             is_root_device: false,
             partuuid: None,
+            cache_type: CacheType::Unsafe,
```

Comment:
Because the purpose of this test is to ensure a request to add a second root device will fail, it does not test any block capability.

The parsing side of this is tested in the `api_server`.